CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Changed
+- `Phlib\JobQueue\AwsSqs\JobQueue` optionally accepts a `$groupKey` parameter
+  - When set and is a valid array key or property name on a job's body, it will be used to set `MessageGroupId` on the message based on the corresponding value in the job's body.
```

Comment:
This is more Readme than Changelog. I suggest cutting the Changelog down to just:

> `AwsSqs\JobQueue` optionally accepts a `$groupKey` parameter to set `MessageGroupId` on the message.

... and then put the more verbose detail into `Readme.md`.